### DISCUSSION TITLE
[Q&A] Snmp on Computers and Appliances

### DISCUSSION THREAD

**Discussion by nykroy** [glpi-project/glpi-agent#355] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

There are any option to inventory with snmp computers an Appliances elements?? the server only accepts for the snmp inventory tasks network devices or  printers 
![image](https://user-images.githubusercontent.com/7745881/223497324-8d6e140e-f84f-4efb-a7c7-9e0d8afbae3d.png)

thanks in advance

**Comment by g-bougard** [comment]
Hi @nykroy 

no for computers, they can't be inventoried this way as too much essential information is simply missing.

For Appliances, we still support some via the [MibSupport/LinuxAppliance](https://github.com/glpi-project/glpi-agent/blob/develop/lib/GLPI/Agent/SNMP/MibSupport/LinuxAppliance.pm) module. And we can extend this module for any new appliances device providing enough information. We also have few support for BSD based Appliances and this can also be extended.
You may check the [Changes](https://github.com/glpi-project/glpi-agent/blob/develop/Changes) file to see is your Appliances device has been added in the past.

**Reply by nykroy** [reply]
No, I need inventory nutanix and logpoint how can i add those elements?

**Reply by g-bougard** [reply]
About Nutanix, a dedicated GLPI plugin is available if you [own a GLPI subscription](https://glpi-project.org/prices/). Search "nutanix" string in this few month ago blog post: https://glpi-project.org/glpi-news-and-work-in-progress-fall-2022/
Few people are still using this plugin in production.

About LogPoint, I'm not aware of any solution. Teclib can probably develop a solution if your company can invest.

You should [contact us directly](https://glpi-project.org/contact/) if you need to discuss about a professional solution.
